spec/support/mockers/mocker.rb

```diff
@@ -0,0 +1,69 @@
+require 'rspec/mocks/standalone'
+
+class Mocker < RSpec::Mocks::InstanceVerifyingDouble
```

Comment:
Makes sense, I think that was the appropriate choice given the tradeoff between adding a new dependency and maintaining the code ourselves for changes of this scale. Re: documentation, I'm assuming that's still pending?